sig-network/README.md

```diff
@@ -25,6 +25,25 @@ Covers networking in Kubernetes.
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-network)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fnetwork)
 
+## Subprojects
+
+The following subprojects are owned by this SIG:
```

Comment:
The subprojects I suggest are:
services, including kube-proxy
ingress
kube-dns
external-dns
pod networking: CNI, ip-masq-agent, etc.
network policy

Reply:
I need OWNERS files to accompany these, I'll take a best guess for now